docs/tools/toolchains/export_to_third_party.md

```diff
@@ -5,11 +5,11 @@ If you'd like to develop on Arm Mbed OS with a third party tool, or migrate to o
 - Keil uVision5.
 - IAR Systems.
 - Make using:
-    - GNU Arm Embedded Toolchain.
+    - GNU Arm Embedded Toolchain 6 (6-2017-q1-update).
     - Arm Compiler 5.
```

Comment:
It's not just 5, it's 5.06 update 6.

Reply:
@cmonr @timurh01 - ARM Compiler 5 supported and in CI is ARM_Compiler_5.06u5 or ARM_Compiler_5.06u6

Reply:
> It's not just 5, it's 5.06 update 6.

Done